src/pages/Dashboard/Dashboard.tsx

```diff
@@ -56,10 +55,40 @@ const DashboardPage = (props: DashboardProps) => {
   const outlineSearch = <MdOutlineImageSearch style={style} />;
   const magicWand = <FaMagic style={style} />;
 
-  const buildVersion = preval`
-    const { execSync } = require('child_process')
-    module.exports = execSync('npm run -s print-version', {encoding: 'utf-8'})
-  `;
+  const buildVersion =  preval(`
+    const execSync = require('child_process').execSync;
+    const shell = require('shelljs');
+
+    const getTodaysDate = () => {
+      const today = new Date();
+      const yyyy = today.getFullYear().toString();
+      const mm = String(today.getMonth() + 1).padStart(2, '0');
+      const dd = String(today.getDate()).padStart(2, '0');
+      return [yyyy, mm, dd].join("");
+    }
+
+    const diff = () => {
+      // Run external tool synchronously
+      const result = shell.exec('git diff --quiet src/').code
```

Comment:
Prefer not to use dependency if the standard library function is easy enough to use.

```javascript
child_process.spawnSync('git', ['diff', '--quiet', 'src/']).status
```

Reply:
Thanks for the update, I will make the requested changes